packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -20,16 +20,17 @@ import {
   useDirtyCheck,
   useConfirmOnLeaving,
 } from '@openmsupply-client/common';
-import { InvoiceLine } from '../../../../types';
+import { InboundLineEditPanel } from './InboundLineEditPanel';
 import { QuantityTable, PricingTable, LocationTable } from './TabTables';
 import { InboundLineEditForm } from './InboundLineEditForm';
 import {
   useInboundLines,
   useInboundFields,
   useSaveInboundLines,
   useNextItem,
+  InboundLineFragment,
 } from '../../../api';
-import { InboundLineEditPanel } from './InboundLineEditPanel';
+import { DraftInboundLine } from '../../../../types';
```

Comment:
The `DraftInboundLine` is extending a fragment `DraftInboundLineFragment`, which is the gql fragment of only the fields needed in this modal - this makes it easier to create 'draft' lines from scratch (add batch button) without needing *all* the fields which aren't even required